test/runtests.jl

```diff
@@ -352,8 +335,21 @@ end
     @test report === NamedTuple()
 end
 
+@testset "Categorical predictors" begin
+    X = (x1=[1.,2.,3.], x2 = categorical([0,1,0]))
+    y = categorical([false, false, true])
```

Comment:
Can we please also have a column that is ordered, as in `x3 = categorical([1, 0, 1], ordered=true)`?